src/NServiceBus.AcceptanceTests/Routing/When_publishing_with_overridden_local_address.cs

```diff
@@ -8,7 +8,7 @@
 
     public class When_publishing_with_overridden_local_address : NServiceBusAcceptanceTest
     {
-        [Test, Explicit("This test fails against RabbitMQ")]
+        [Test]
```

Comment:
Removed the explicit since @Particular/rabbitmq-transport-maintainers can just add Assert.Ignore to their configure test execution if that test really fails. 

With the learning transport this test is green with or without overriding the local address